src/ngx_http_lua_ssl_certby.c

```diff
@@ -880,6 +880,30 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 #endif
 }
 
+int
+ngx_http_lua_ffi_ssl_client_addr(ngx_http_request_t *r, char **addr,
```

Comment:
I first name it `raw_client_addr`, I am returning ip address in plain text, like `ngx.var.remote_addr`, not `ngx.var.binary_remote_addr`, so I name it `client_addr`, am I correct?